pkg/subctl/cmd/createbroker.go

```diff
@@ -80,5 +84,35 @@ var createBrokerCmd = &cobra.Command{
 			panic(err.Error())
 		}
 		fmt.Printf("There are %d pods in the cluster\n", len(pods.Items))
+
+		// Create CRDs
```

Comment:
I would isolate all the CRD creation/verification to it's own pkg, like pkg/broker or something like that, in a way that later on it's easy to move to the operator instead of subctl without moving chunks of code.

Reply:
Very good idea, I was thinking about early things along these lines, that I wanted to next find a way to structure the logic to make it more logically distinct. I'll take a stab at this pkg approach, thanks!